app/models/show.rb

```diff
@@ -3,6 +3,8 @@ class Show < ActiveRecord::Base
   REGULAR_SHOW = 'Regular Show'
   TYPES = [REGULAR_SHOW, 'Special Event', 'Class', 'Subscription']
 
+  REMINDERS = ['Never', '12 hours', '24 hours', '36 hours', '48 hours']
```

Comment:
Shouldn't the reminder options be more verbiage per the customer meeting? i.ee.

"12 hours before showdate time"
"24 hours before showdate time"
"36 hours before showdate time"
"48 hours before showdate time"